modules/core/src/lib/load.js

```diff
@@ -27,6 +27,9 @@ export async function load(url, loaders, options) {
   }
 
   // at this point, data can be binary or text
-  const response = await fetchFile(url, options);
-  return parse(response, loaders, options, url);
+  let data = url;
+  if (typeof data === 'string') {
```

Comment:
We may need to update the reference-api doc for `load`. Maybe just say if the first parameterisn't a string, it is passed directly to parse?